qiita_pet/templates/edit_study.html

```diff
@@ -237,12 +237,15 @@
     {% set kwargs = {'tabindex': idx, 'class_': 'form-control', 'required': form_item.flags.required} %}
     {% set additional_info = '' %}
     {# form-item-specific rendering #}
+    {% if form_item.label.text == 'Study Title' %}
+      {% set additional_info = '<u>Recommended format:</u><br/>PI last name brief project description<br/>For example: <i>Darwin Galapagos Finch Feces</i><br/><br/>' %}
```

Comment:
I can add the PI to the search though and that will make it easier on everyone. To me it makes more sense to do `PI = Rob Knight` than 'study_title includes Knight`.